cmd/influxd/server_integration_test.go

```diff
@@ -1389,7 +1390,31 @@ func Test3NodeServer(t *testing.T) {
 
 	runTestsData(t, testName, nodes, "mydb", "myrp", len(nodes))
 	runTest_rawDataReturnsInOrder(t, testName, nodes, "mydb", "myrp", len(nodes))
+}
+
+func Test3NodeServerFailover(t *testing.T) {
+	testName := "3-node server integration"
 
+	if testing.Short() {
+		t.Skip(fmt.Sprintf("skipping '%s'", testName))
+	}
+	dir := tempfile()
+	defer func() {
+		os.RemoveAll(dir)
+	}()
+
+	nodes := createCombinedNodeCluster(t, testName, dir, 3, nil)
+
+	// allow cluster to stabilize
```

Comment:
Is this needed? If so, should we add it to all other cluster tests in this suite? A different PR, though, if needed.